modules/ocf_www/manifests/site/ocfweb_redirects.pp

```diff
@@ -2,9 +2,11 @@
 # redirect those sites to the appropriate pages on ocfweb.
 class ocf_www::site::ocfweb_redirects {
   # accounts
-  $accounts_canonical_url = $::hostname ? {
-    /^dev-/ => 'https://dev-accounts.ocf.berkeley.edu/',
-    default => 'https://accounts.ocf.berkeley.edu/',
+  if $::dev_config {
+    $accounts_canonical_url = 'https://dev-accounts.ocf.berkeley.edu/'
+  }
+  else {
+    $accounts_canonical_url = 'https://accounts.ocf.berkeley.edu/'
```

Comment:
I think keeping these as selectors makes the most sense, or at least keeps it cleaner since you don't have to duplicate the variable name:

```puppet
$accounts_cannonical_url = $::dev_config ? {
  true    => 'https://dev-accounts.ocf.berkeley.edu/',
  default => 'https://accounts.ocf.berkeley.edu/',
}
```

Reply:
Huh, I've never thought to do that before, kinda neat. Alternatively you could do
```puppet
$accounts_cannonical_url = if $::dev_config {
  'https://dev-accounts.ocf.berkeley.edu/'
} else {
  'https://accounts.ocf.berkeley.edu/'
}
```
No definite opinion on which looks better.

Reply:
Yeah, the if statement version is ok, I think the indentation makes it a little less clear, but I don't have a strong opinion on one over the other either.